README.md

```diff
@@ -92,7 +84,7 @@ To run a Buddy instance from the command line interface (CLI), use the following
   --help                 display this help message
   --telemetry-period=[N] set period for telemetry when we do snapshots
   --disable-telemetry    disables telemetry for Buddy
-  --threads=[N]          start N threads on launch, default is 4
+  --threads=[N]          start N threads on launch, default is CPU count
```

Comment:
May be confusing as a CPU usually consists of 2 or more cores. So "CPU core count" may be more correct if you mean that.